pkg/trantor/moduleconfig.go

```diff
@@ -17,6 +18,7 @@ type ModuleConfig struct {
 	BatchDB       t.ModuleID
 	BatchFetcher  t.ModuleID
 	Checkpointing t.ModuleID
+	CVC           t.ModuleID
```

Comment:
Same as previous comment. `ChkpValidator`? (or something similar, but more descriptive than CVC?) (not a strong opinion)

Reply:
sure, will do!

Reply:
done